### PR TITLE
[FIX] Missing Capital (CALL) infos

### PR DESCRIPTION
**Pull request by capital-technologies**
**Project Name:** **Capital Technologies & Research**
**Website:** **https://www.mycapitalco.in**
**Description:** Capital Technologies & Research will develop a decentralized private communication system where users can easily enjoy their private communications.
**Pull Requests:**
- [X] **[152](https://github.com/jl777/coins/pull/152)@jl777/coins**
- [x] **[5](https://github.com/pbca26/agama-wallet-lib/pull/5)@pbca26/agama-wallet-lib**
- [x] **[100](https://github.com/KomodoPlatform/Agama/pull/100)@KomodoPlatform/Agama/routes/ports.js@dev**
- [x] **[72](https://github.com/KomodoPlatform/EasyDEX-GUI/pull/72)@KomodoPlatform/EasyDEX-GUI**

### CODE REVIEW

File: app/config.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ const defaults = {
 		'SUPERNET',
 		'VTC',
 		'ZEC',
+		'CALL',
```
Comment (lukechilds) [inline app/config.js]: Can you remove this line, it shouldn't be added as a default coin.

File: app/renderer/assets/custom-cryptocurrency-icons/call.svg [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
...dgmwl3r+cBhebLdZdpnnetgiTRiT6t6rlbna94I4a3G+ybwgJeV2yOW67/AYXW4DfCAkFZ7S6iDKnwWi9EkRbXOuuCKPAYkz3nt2mynCHvOY0eYdJsoSEak499dDLLcFTjG6tQmFMvdtLWb6Kb/gCJuLf3HRp9o3hKdkKDDfdwxY1yO0Z9lmFTJTfCgTeV2lwxPtI691ji9UoVyjNW80jwNOyJFrpbZ3qBN/XfJ/ademCD4yv10bDFnsadsnfqcXyCJfWMFmusS1ws5zwRe8obcl3WWIxYjxUL+R1XnDIq8EiEyPeptTxbmZ4y3aPNrZ+XxLZfuQ7BgZvXa1wOrio2Chimmhqo2sNt8FJ022NiBrZXUel2G+UjwLTZILxPoy4VtLHIzI9o8ho1+ituxQJop1vwnAZL9OT4asna0wQL76xHaAGn3WLaTGe0NnZJgIPblYRbPH58j0bTh/lGlR6VryuOkqTiAoXJOuAkE2XsD0ly5euQnwQ8LPGuJYS4Cs++QynX4r5nlaJaeLC2sp0gz13CX9nkk466mLPJQhMlO6IseL0tFCZuy4n7PvS+OxbLNMDV+Gs8MZ3p3ubHJ6fhS4WWqS/llxXbQE6uFsaxuuCFPeaf0Xt5blH4dUXe5qfhlfvMW7AXPT2oOsvo/Y1pje4qlWLhMuoH+BKrqQP9aAce73hJaWmOu64OLd4NuK+ZA3iJEsT5awz8oxUrYeJ9nj4yo9IV/pPAVLNlCfDRnv9yirP2ueE3pIkSxAvQYxoF5x0QoZ8A2U4bLVP7Wodm8eV/1cDokw3zwz9cZ/XJOgsw3G7VevotKOSXKuzammOeD/iStUEW6/0tNFyAlmyZEqX4LxTdujvF3bbrcJ2f3UA33bO7/TTT64Ye+oYzEk13yl7nalr52krAmON8qkiBxtc15xohhGKDDcS/+0PquwOQhguIs94pUL6WtkwkLstCBRIV+aCdOmqlTnmSB079h3+VQ98aFQjtUeboNpeMV5qYBFvIwK1SNFDpgzJYlGq1AUXHLZPgl8pt8dARZ6wK6LOQ4ZZ6bC/t8Y0bo1JXIt0ncXJkClVjKNKHBFlrLlyrPSoIZZaY5BOfh1ce2sFtCaBSMTLli5RtfOijDdND2PAI837rwWfD8RKsjhwEU288ubaC48p8kJrK2f/DxHsbytN+5VBAAAAAElFTkSuQmCC"/>
```
Comment (lukechilds) [inline app/renderer/assets/custom-cryptocurrency-icons/call.svg]: This isn't an SVG it's just a PNG with an SVG wrapper. We require vector graphics to support high DPI displays.

File: app/renderer/components/CurrencyIcon.js [hunk 3]
```diff
@@ -17,6 +17,7 @@ const hasCustomIcon = new Set([
 	'SUPERNET',
 	'WLC',
 	'MYTH',
+	'CALL',
```
Comment (lukechilds) [inline app/renderer/components/CurrencyIcon.js]: Would you mind submitting an issue to create your icon over at https://github.com/atomiclabs/cryptocurrency-icons?

We prefer to keep all the icons over there, we just use custom assets for a few Komodo Asset Chains.

File: app/marketmaker/supported-currencies.js [hunk 4]
```diff
@@ -401,6 +401,20 @@ const supportedCurrencies = [
...
+		etomic: '0x2c02c801a3324b371ad285c64af372ff1e23f460',
+		rpcport: 80,
+	},
+	{
+		coin: 'CALLG',
+		name: 'Capital GAS',
+		decimals: 18,
+		etomic: '0x6c28f85eb1dbc7651ce02f7af1f29890ad084b94',
+		rpcport: 80,
+	},
```
Comment (lukechilds) [inline app/marketmaker/supported-currencies.js]: Are you able to add these alphabetically like the other currencies?

**Comment by capital-technologies** [comment]
We have fixed all issues so far. We will submit another PR in the future with a valid SVG file. As a bonus, we have added Romanian (ro) native translation.

**Comment by sindresorhus** [comment]
> As a bonus, we have added Romanian (ro) native translation.

We appreciate that, but that should be done in a separate PR.

**Comment by capital-technologies** [comment]
@sindresorhus We have removed the translation files in this PR

**Comment by lukechilds** [comment]
Thanks for the contribution @capital-technologies looking forward to those follow-up PRs 🎉

**Pull request merged**